### PR TITLE
Space cash descriptions are more descriptive

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
tweak: Stacks of space cash now tell you their total value and their value per bill when you examine them.
/:cl:

I just thought this would be a nice thing to do.

### CODE REVIEW

**Comment by nicbn** [comment]
Nice! Now it's easier to me to use space cash to... uh... argh.

**Comment by Bawhoppen** [comment]
I'm pretty sure there's some special text that already tells you (coins atleast), test it in game

File: code/game/objects/items/stacks/cash.dm [hunk 1]
```diff
@@ -10,40 +9,64 @@
...
+
+
+/obj/item/stack/spacecash/merge(obj/item/stack/S)
+	. = ..()
+	update_desc()
+
+
+/obj/item/stack/spacecash/examine(mob/user)
+	update_desc()
+	..()
```
Comment (Cyberboss) [inline code/game/objects/items/stacks/cash.dm]: This one shouldn't be necessary

**Review by Cyberboss — DISMISSED** [review]

File: code/game/objects/items/stacks/cash.dm [hunk 2]
```diff
@@ -10,40 +9,63 @@
...
+	var/total_worth = amount*value
+	desc = "It's worth [total_worth] credit[( amount > 1 ) ? "s" : ""]"
+
+
+/obj/item/stack/spacecash/merge(obj/item/stack/S)
+	. = ..()
+	update_desc()
+
+/obj/item/stack/spacecash/use()
+	..()
```
Comment (lzimann) [inline code/game/objects/items/stacks/cash.dm]: `. = ..()` to be safe

**Pull request merged**